ios/FluentUI.Demo/FluentUI.Demo/Demos/HUDDemoController.swift

```diff
@@ -6,134 +6,137 @@
 import FluentUI
 import UIKit
 
-class HUDDemoController: DemoController {
-    override func viewDidLoad() {
-        super.viewDidLoad()
+class HUDDemoController: DemoTableViewController {
```

Comment:
Should we add an extension that conforms to DemoAppearanceDelegate to show the theme and control overrides?

Reply:
Yes! It's a good idea to have it tested as well.